runtime/moonbase/tests/xcm_mock/parachain.rs

```diff
@@ -586,7 +591,23 @@ impl Default for AssetType {
 
 impl From<MultiLocation> for AssetType {
 	fn from(location: MultiLocation) -> Self {
-		Self::Xcm(location)
+		match location {
+			// Change https://github.com/paritytech/cumulus/pull/831
+			// This avoids interrumption once they upgrade
+			// We map the previous location to the new one so that the assetId is well retrieved
+			MultiLocation {
+				parents: 1,
+				interior: X2(Parachain(id), GeneralIndex(index)),
+			} if id == StatemintParaId::get() => Self::Xcm(MultiLocation {
+				parents: 1,
+				interior: X3(
+					Parachain(id),
+					PalletInstance(StatemintAssetPalletInstance::get()),
```

Comment:
This seems safe for now. If Kusama were to change their pallet instance (very unlikely) or add a second instance of Statemint (unlikely), this would do bad things, but we will be able to remove this before then anyway -- right?

Reply:
Yes, the idea is that we remove this once Statemine has upgraded